pkgs/applications/networking/mailreaders/caeml/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+buildGoModule rec {
+  pname = "caeml";
+  version = "unstable-2023-05-24";
+
+  src = fetchFromGitHub {
+    owner = "ferdinandyb";
+    repo = pname;
```

nitpick: 
```suggestion
    repo = finalAttrs.pname;
```
and remove rec

---

looks like `buildGoModule` doesn't support the `finalAttrs` pattern yet :/

---

Yeah sorry, didn't know that :see_no_evil: